src/wkok/openai_clojure/azure/api.clj

```diff
@@ -0,0 +1,38 @@
+(ns wkok.openai-clojure.azure.api
+  (:require [wkok.openai-clojure.azure.core :as core]))
+
+
+(defn create-completion
```

Comment:
I kept the naming of the functions from  `wkok.openapi_clojure,api`
even though the operation name is slightly different